.gitlab-ci.yml

```diff
@@ -394,9 +394,9 @@ OSTree simplified installer:
   parallel:
     matrix:
       - RUNNER:
-          - rhos-01/rhel-8.7-nightly-x86_64
+          - rhos-01/rhel-8.8-nightly-x86_64
           - rhos-01/centos-stream-8-x86_64
-          - rhos-01/rhel-9.1-nightly-x86_64
+          - rhos-01/rhel-9.2-nightly-x86_64
```

Comment:
why are we also dropping 9.1/8.7? shouldn't we just add to the list here?

Reply:
We're dropping the nightly tests on these. At the time of writing there was no 9.1 and 8.7 GA. 

I've updated this PR to switch 8.6-ga and 9.0 GA to the next version.